art_brain/src/node.py

```diff
@@ -22,21 +22,22 @@ class ArtBrain:
     NOP = -1  # no operation
     GET_READY = 0  # retract arms etc.
     MANIP_PICK = 1
-    MANIP_PLACE = 2  # TODO proc nepouzit ProgramItem.MANIP_PLACE?
```

Comment:
Ha, tohle vrací nějaké úpravy formátování, které jsem v brainu udělal. Neprojde to přes travis :) Před inline komentářem musí podle pep8 být dvě mezery.